Makefile

```diff
@@ -1,4 +1,4 @@
-CC=/usr/bin/gcc
+CC=gcc
```

Comment:
Why was this necessary?

Reply:
Just realized I could just do it in flake.nix (makeFlags) instead of the Makefile.


The only file in /usr/bin on NixOS is `env`, which is why I had to use `/usr/bin/env bash`.

However, `env` doesn't exist in /usr/bin within the Nix build sandbox, as it would cause impurities apparently (Nix likes to be very reproducible).